manual/Website2/package.json

```diff
@@ -41,15 +45,14 @@
     "crc-32": "^1.2.0",
     "cross-env": "^7.0.0",
     "html-webpack-plugin": "^4.0.0-beta.8",
-    "mini-css-extract-plugin": "^0.6.0",
     "moment": "^2.24.0",
     "optimize-css-assets-webpack-plugin": "^5.0.1",
-    "react": "file:../../node_modules/react",
-    "react-dom": "file:../../node_modules/react-dom",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
```

Comment:
don't need file paths, yarn workspaces is helping us dedupe

Reply:
not a yarn workspace tho